cypress/support/step_definitions/dictionary/addBulkConceptsToDictionary/addBulkConceptsToDictionary.ts

```diff
@@ -0,0 +1,128 @@
+/// <reference types="cypress" />
+/// <reference types="../../../" />
+import { 
+  After, 
+  Before, 
+  Given, 
+  Then, 
+  When } from "cypress-cucumber-preprocessor/steps";
+import { getDictionaryId, getUser, getConceptId } from '../../../utils';
+
+Given("the user is on the view dictionary concepts page", () => {
+  cy.visit(`/users/${getUser()}/collections/${getDictionaryId()}/concepts/`);
+  cy.url().should(
+    "contain",
+    `/users/${getUser()}/collections/${getDictionaryId()}/concepts/`
+  );
+});
+
+Given('the user is on the "Add concepts in bulk from CIEL" page', () => {
+cy.visit(
+`/users/${getUser()}/collections/${getDictionaryId()}/add/?fromSource=CIEL/`
+);
+cy.url().should(
+"contain",
+`/users/${getUser()}/collections/${getDictionaryId()}/add/?fromSource=CIEL/`
+  );
+});
+
+Given('CIEL concept "1000" is already in the dictionary', () => {
+  cy.visit('/actions/');
+  cy.url().should(
+    "contain", '/actions/'
+  );
+});
+
+When('the user clicks the "Add concepts" button', () => {
+  cy.findByText("Add concepts").click({force: true});
+});
+
+When('the user selects "Add bulk concepts"', () => {
+cy.visit(
+  `/users/${getUser()}/collections/${getDictionaryId()}/add/?fromSource=CIEL/`
+  );
+});
+
+When(
+  /the user enters concept Id "(1000|1001|1002)"/, () => {
+    cy.getConcept("#conceptId")
+    cy.get("textarea")
+  }
+); 
+
+When('the user re-enters concept Id "1000"', () => {
+  cy.getConcept("#conceptId")
+  cy.get("textarea")
+});
+
+When("the user is send to progress notifications page", () => {
```

Comment:
```suggestion
When('the user is send to "Progress notifications" page', () => {
```

Reply:
"send" -> "sent". However, this should probably be "the user navigates to the progress notifications page" or similar. Note, too, that there's likely to be a delay. Rather than jumping over to another screen, it might be better just to verify that the expected concepts are added to the dictionary. We can have some separate tests for what ought to be on the notification screen.